CONTRIBUTING.md

```diff
@@ -40,8 +40,11 @@ The following is a short list of commands that can be run in the root of the pro
 - Run `make` see a list of available targets.
 - Run `make test` to run all unit tests.
 - Run `make lint` to validate against our linting rules.
-- Run `make e2e` will run end to end tests against an Atlas instance,
-  please make sure too have set `MCLI_*` variables pointing to that instance.
+- Run `export E2E_TAGS=e2e,atlas && make e2e-test` will run end to end tests against an Atlas instance,
+  please make sure to have set `MCLI_*` variables pointing to that instance.
+- Run `export E2E_TAGS=e2e,cloudmanager && make e2e-test` will run end to end tests against an Cloud Manager instance.<br />
+  Please remember to update `the e2e/cloud_manager/e2e.env` with the name of the host that is running your automation agent and 
+  to set `MCLI_*` variables to point to your Cloud Manager instance. 
```

Comment:
not for this pr but for when you start using the `e2e.env` file, we should probably start using something like https://github.com/joho/godotenv end put all those env there instead of forcing users to export them

Reply:
I have created this task for that [CLOUDP-64365](https://jira.mongodb.org/browse/CLOUDP-64365)